dnssec_keyscan.go

```diff
@@ -46,15 +51,15 @@ func (k *DNSKEY) ReadPrivateKey(q io.Reader, file string) (PrivateKey, error) {
 		}
 		priv.PublicKey = *pub
 		return (*DSAPrivateKey)(priv), e
-	case "1 (RSAMD5)":
+	case RSAMD5:
 		fallthrough
-	case "5 (RSASHA1)":
+	case RSASHA1:
 		fallthrough
-	case "7 (RSASHA1NSEC3SHA1)":
```

Comment:
Other cases would have to be updated as well to match BIND9 names then. This approach is more error proof as it doesn't care what the human readable version of the string says. This is how BIND9 parses it's own files as well (from what I saw in code, didn't test).